docs/upctl_ip-address_assign.md

```diff
@@ -0,0 +1,35 @@
+## upctl ip-address assign
+
+Assign an ip address
+
+```
+upctl ip-address assign [flags]
+```
+
+### Options
+
+```
+      --access string   Is address for utility or public network. (default "public")
+      --family string   The address family of new IP address. (default "IPv4")
+      --server string   The server the ip address is assigned to.
+      --mac string      MAC address of server interface to assign address to. Required for non-floating addresses.
+      --zone string     Zone of address, required when assigning a detached floating IP address.
+      --floating        Whether the address to be assigned is a floating one.
```

Comment:
this is actually a bit confusing now, i think? 

Broader question: should we, before 1.0, have another go at the 'tri-state' boolean flags, (eg. true, false or not set) and turning them into --enable-floating / --disable-floating? I think this was discussed earlier on and there was even a bit of poc code floating around in the discussion.

Reply:
I don't think there are tree states as the struct get initalized with a False value, https://github.com/UpCloudLtd/upcloud-go-api/blob/master/upcloud/ip_address.go#L52